.github/workflows/integration.yml

```diff
@@ -135,8 +135,7 @@ jobs:
           path: ./agoric-sdk
       - uses: ./agoric-sdk/.github/actions/restore-node
         with:
-          # XXX loadgen not compatible with 18.19 https://github.com/Agoric/agoric-sdk/pull/8365#issuecomment-1848003597
```

Comment:
pretty sure this was legit https://github.com/Agoric/agoric-sdk/actions/runs/11863590746/job/33065361760?pr=10500

Reply:
Damn. Then I guess the right fix is instead downgrading execa to e.g. ^9.3.1?

Reply:
This is the fix I'd really like to see =)
 - https://github.com/Agoric/testnet-load-generator/issues/117

But downgrading execa makes sense for now.